### PR TITLE
initializing fastlane structures

### PR DESCRIPTION
**Pull request by IzzySoft**
[Fastlane](https://fastlane.tools/) somehow got the standard for metadata of Android and iOS apps. It's used a.o. by [F-Droid](https://f-droid.org/), by [my repo](https://apt.izzysoft.de/fdroid/index/apk/com.isaiahvonrundstedt.fokus) – but also to deploy to Google Play. Thought to get you started with it, and hope you find this "minimal kit" useful.

You might wish to move other metadata into this structure as well, e.g. your screenshots from the `art` directory. For a quick guide, be welcome to use [my cheat sheet](https://gitlab.com/snippets/1895688).

### CODE REVIEW

**Review by asayah-san — APPROVED** [review]

**Comment by IzzySoft** [comment]
Thanks for merging! Should you move the screenshots in or add e.g. changelogs, please let me know so I tell my updater to fetch them as well.

**Comment by 7RST1** [comment]
Randomly noticed while checking language code for Norwegian;
Shouldn't the German language code be de-DE instead, or doesn't it matter? Saw the code [here](https://docs.fastlane.tools/actions/deliver/#available-language-codes). I know nothing about fastlane, just happen to notice the difference in language code.

**Comment by IzzySoft** [comment]
> Shouldn't the German language code be de-DE instead

If it's specific to the German as spoken in Germany (as opposed to e.g. de-AT spoken in Austria), yes. But that would mean our Austrian fellows would fall back to en-US unless they explicitly specified they wanted to have de-DE. Usual browser setups for a German speaking user in Austria are rather "de-AT, de, en". Hence, as no language specific to any given region was used, only "de" is fine – and our Austrian friends could supply de-AT in addition if they feel like (similar to other de-*).

**TL;DR:** just using "de" makes it eligible for all German speaking countries. Specifying the region as well (de-DE) would not. "en-US" is a stupid exception as that's the fall-back for many tools (who ever made it that should be punished; "en" would have been much more sane).

> Saw the code here

which is totally incomplete, if I might say so, and thus rather gives examples. What stands behind those codes is [ISO-639](https://en.wikipedia.org/wiki/ISO_639_macrolanguage) and [ISO-3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) (see the alpha-2 code column there), also consult [Language code](https://en.wikipedia.org/wiki/Language_code) (the second row in that table, "IETF language tag", sums it up). So basically, it's `<language>[-<region>]`, with the region being optional.
